data/guests/jturnbull2.yml

```diff
@@ -0,0 +1,5 @@
+Name: jturnbull
+"FullName": "James Turnbull"
+"Bio": "James Turnbull is the author of seven technical books about open source software, including a book on Docker. He was previously the CTO at Kickstarter and in leadership roles at Docker, Venmo and Puppet. James speaks regularly at conferences including OSCON, Velocity, Linux.conf.au, FOSDEM and a number of others. He is a past president of Linux Australia, a former committee member of Linux Victoria, was Treasurer for LCA 2008, and serves on the program committees of LCA and OSCON. James serves as the co-chair of O'Reilly's Velocity conference and on the program committees of Linux.conf.au and OSCON."
```

Comment:
Sorry! One more - this can be deleted: ", and serves on the program committees of LCA and OSCON."